Project1/PathMap.h

```diff
@@ -3,15 +3,16 @@
 #include "CellPath.h"
 #include <map>
 
-using CostCell = logic_pair<int, CellRef>;
+
 //TODO(Torrey): rename this class
-template <typename grid>
+template <typename grid, typename cell>
```

Comment:
I feel like there is a better way than also having the cell in the template, but I don't know how